api/bootstrap.proto

```diff
@@ -220,11 +220,23 @@ message StatsdSink {
   }
 }
 
+// Stats configuration proto schema for built-in *envoy.dog_statsd* sink.
+// The sink emits stats with `DogStatsD <https://docs.datadoghq.com/guides/dogstatsd/>`_
+// compatible tags. Tags are configurable via :ref:`StatsConfig <envoy_api_msg_StatsConfig>`.
+message DogStatsdSink {
```

Comment:
We probably should have a `api/stats` directory and move the pluggable stats to their own protos there. Same goes for `api/trace`. Doesn't have to be done in this PR, but if there is agreement amongst maintainers, we can file an issue.

Reply:
+1. Probably in `api/stats` or `api/stats_sink`?

Reply:
+1, do you mind just doing that in this PR?

Reply:
Sure. Done in 2acb78bf0b8c87b18e9fbbf15cd6a29c66e62405. If it's OK, I'll do the same thing for `api/trace` in another PR.